### ISSUE TITLE
train it for 2class and the loss-c is decreased but loss-l is still not. What's wrong? 

### ISSUE THREAD

**Issue by haochange** [ruinmessi/RFBNet#73]
My hyperparameter is: CUDA_VISIBLE_DEVICES=0,1 python train_RFB.py -v RFB_E_vgg -s 512 -d VOC --basenet ./weights/vgg16_reducedfc.pth  --jaccard_threshold 0.5 --batch_size 32 --num_workers 8 --cuda True --ngpu 2 --lr 4e-3 --momentum 0.9 --resume_net None --resume_epoch 0 --max_epoch 300 --weight_decay 5e-4 --gamma 0.1 --log_iters True --save_folder ./training_model/RFB_E_vgg/
Train set includes 2,500 images,and i only changed the num_class from 21 to 2 in the train_rfb.py and voc0712.py.

**Comment by Z0org** [comment]
I got the same problem. Did yo solve it or have a workaround?

**Comment by lzl4525** [comment]
I got the same problem,how to fix it
